backend/core/serializers.py

```
@@ -993,9 +993,40 @@ class Meta:
+        [
+            "id",
+            "min_score",
+            "max_score",
+            "implementation_groups_definition",
+            "ref_id",
+            "reference_controls",
+        ]
+    )
+
+    class Meta:
+        model = Campaign
```

Ruff (0.11.9)

`Campaign` may be undefined, or defined from star imports

(F405)